CorsixTH/Lua/hospital.lua

```diff
@@ -2365,6 +2365,16 @@ function Hospital:getRandomBusyRoom()
   if #chosen_room.door.queue >= busy_threshold then return chosen_room end
 end
 
+--! Open the hospital
+function Hospital:open()
+  self.opened = true
+end
+
+--! Check if the hospital is ready to accept patients
+function Hospital:canAcceptPatients()
+  return self:hasStaffedDesk() and self.opened
```

Comment:
```suggestion
  return self.opened and self:hasStaffedDesk()
```
Do the cheap check first to avoid wasting CPU cycles.

Reply:
I agree, checking a single variable is better.